truffle/contracts/Migrations.sol

```diff
@@ -1,16 +1,16 @@
 // SPDX-License-Identifier: MIT
-pragma solidity >=0.4.21 <0.7.0;
+pragma solidity >=0.4.22 <0.9.0;
 
 contract Migrations {
-  address public owner;
+  address public owner = msg.sender;
   uint public last_completed_migration;
 
   modifier restricted() {
-    if (msg.sender == owner) _;
-  }
-
-  constructor() public {
-    owner = msg.sender;
+    require(
+      msg.sender == owner,
+      "This function is restricted to the contract's owner"
+    );
+    _;
   }
 
   function setCompleted(uint completed) public restricted {
```

Comment:
Should we remove this file completely? Up for debate but we discussed all future boxes not including this anymore as its not really used and tends to waste gas for new users. Your call though.

Reply:
Yea this should probably go. I believe @gnidan removed this along with its migration file in past Truffle demos.

Reply:
And yea I want to keep the test directory to reflect a full Truffle project structure.

Reply:
Resolved in [this commit](https://github.com/truffle-box/react-box/commit/aff17a2efc556030b0ad91ebe443e7663ba45b98)!